vidarr-cromwell/src/main/java/ca/on/oicr/gsi/vidarr/cromwell/CromwellOutputProvisioner.java

```diff
@@ -116,10 +109,17 @@ private void check(
                 monitor.storeDebugInfo(result.debugInfo());
                 switch (result.getStatus()) {
                     // In the case of failures ("Aborted" or "Failed"), request the full metadata
-                    // from Cromwell
+                    // from Cromwell if we don't already have it
                     // so we can have call info for debugging.
                   case "Aborted":
                   case "Failed":
+                    if (debugCalls){
+                      monitor.log(
+                              System.Logger.Level.INFO,
+                              "Cromwell OutputProvisioner already configured to fetch calls info. Skipping second request."
```

Comment:
This log line to me reads like a statement that the `check` call is trying to change the CromwellOutputProvisioner configuration but is unsuccessful in doing so... I'm not sure I'd know what information I was supposed to take from this line if I saw it in the logs. What was your thought behind logging here as opposed to `break` with code comment?

Reply:
Line 90 logs "Checking Cromwell job [...]" and line 107 logs "Cromwell job _x_ is in state _y_". If the OutputProvisioner (and WorkflowEngine because it has the same behaviour) _does_ need to make a second request, line 126 logs "Cromwell job _x_ is failed, fetching call info" and then logs whether that request was successful. A Vidarr administrator may not know off the top of their head whether the cromwell plugins have been configured with 'debugCalls' or not. This prevents the logs from looking like nothing happened or something threw a silently handled exception and exited early.

Reply:
What do you think about updating the log message slightly - `Cromwell OutputProvisioner is configured to have already fetched calls info.` ? Or updating the log on line 107 to include information  that the calls were fetched if debugCalls is on? (I'm trying to eliminate the interpretation that something is trying to change the Cromwell OutputProvisioner configuration)

Reply:
Sure, I can reword it.